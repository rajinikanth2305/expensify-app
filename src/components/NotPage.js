import React from "react";
import {Link} from "react-router-dom";
const NotPage=()=>(
    <div>
    
    This is not found help page-<Link to="/">Home</Link>
    </div>

)
export default NotPage