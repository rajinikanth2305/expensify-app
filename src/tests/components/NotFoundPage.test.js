import React from 'react';
import { shallow } from 'enzyme';
import NotPage from '../../components/NotPage';

test('should render not page correctly', () => {
  const wrapper = shallow(<NotPage />);
  expect(wrapper).toMatchSnapshot();
});




//snapshot allows to track changes in data
//renderer function takes render function and it prints the output for component
//we have to install ReactShallowRender for testing the components
//library called nyzyme  for airbnb to actually writing test cases
//step 1 to install  