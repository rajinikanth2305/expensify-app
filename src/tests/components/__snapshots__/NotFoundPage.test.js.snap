// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render not page correctly 1`] = `
<div>
  This is not found help page-
  <Link
    replace={false}
    to="/"
  >
    Home
  </Link>
</div>
`;
